.github/workflows/storybook_deploy.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy Storybook
+
+on:
+  pull_request:
+    paths:
+      - '**[Ss]tories**'
+
+jobs:
+  deploy_storybook:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        clean: false
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Fetch all branches
+      run: 'git fetch --no-tags --prune --depth=1 origin +refs/heads/*:refs/remotes/origin/*'
+    - name: Run npm install
+      run: npm install
+    - name: Build Storybook in temporary folder
+      run: npm run build-storybook -- --output-dir=tmp/storybooks/${{ github.event.pull_request.number }}
+    - name: Checkout gh-pages
+      run: |
+        git checkout gh-pages
+        ls tmp/storybooks/${{ github.event.pull_request.number }}
+    - name: Move files to storybooks/${{ github.event.pull_request.number }}
+      run: |
+        mkdir -p storybooks/${{ github.event.pull_request.number }}
+        rm -rf storybooks/${{ github.event.pull_request.number }}/*
+        mv tmp/storybooks/${{ github.event.pull_request.number }}/* storybooks/${{ github.event.pull_request.number }}
+        ls storybooks/${{ github.event.pull_request.number }}
```

Comment:
We do not see what command is being executed in the console, only the output:

![checkout-pages](https://user-images.githubusercontent.com/2664090/75530104-f6388880-5a13-11ea-809a-ed5cd1fdfeba.png)


It might be useful to add `echo` to commands printing info to the console, I see you do it for `git add .` below.

Reply:
@xprazak2 You can click the little triangle to see the commands, sorry I didn't get a screenshot of it